style.css

```diff
@@ -0,0 +1,229 @@
+body {
+  background-color: #fff;
+  font-family: Helvetica, sans-serif;
+  margin:0;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 30px;
+  text-align: center;
+}
+
+h3 {
+  font-size: 30px;
+  text-align: center;
+}
+
+p {
+text-align: center;
+}
+
+.footer {
+  background-color: black;
+  color: white;
+  margin: 0;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+/*** HEADER ***/
+
+header {
+  background: black;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+}
+
+.hamburger {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+header img {
+  width: 50px;
+  padding: 5px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+
+/*** WELCOME CONTAINER ***/
+
+.welcome-container .container { 
+  padding: 100px 16px;
```

Comment:
This padding creates a horizontal scroll which we want to prevent when working with responsive design.